src/components/DepositWidget/useRowActions.tsx

```diff
@@ -65,8 +66,13 @@ export const useRowActions = (params: Params): Result => {
   }
 
   async function enableToken(tokenAddress: string): Promise<void> {
-    const { symbol } = getToken('address', tokenAddress, balances)
+    const { symbol } = getToken('address', tokenAddress, balances) as Required<TokenBalanceDetails>
+
     try {
+      if (!userAddress) {
+        throw new Error('No logged in user found. Please check wallet connectivity status and try again.')
+      }
```

Comment:
Could this be extracted as a function?

Reply:
tried this before changing it back and pushing as I couldn't figure out how to type the function out - it was:

```
function checkUserAddress(userAddress?: string): void {
  if (!userAddress) {
        throw new Error('No logged in user found. Please check wallet connectivity status and try 
        again.')
      }
}
```

but the code following line 74 was throwing issues with `userAddress` still - I should just write a typeguard function instead. ill take a look

Reply:
type [never](https://www.typescriptlang.org/play/?target=99&ssl=12&ssc=1&pln=12&pc=19#code/FAMwrgdgxgLglgewgAigCwKZQNYFUDOGATgIIAmZRG++AFAJQBcyEGAbsaJLIiulnkKkKVGrTBDylavmb4YROBADmTZGwRwyXaPCSpMOAsSmi6EkyJkB+OQqWrmGrcgA+LdsWQBvYMmRwIMi0AIQWwtI09D5+-nH+MGhECADuHmkAokTJRLQA5AByCMgANgjKyhhkASjhyCAIkGQAdMgACiUYAIaEBgLIKV0lnTCoSKw8bHAwAJ7I8l0wEshdENUKc13KXUrNefSxAL7Ax8BQSPLIian4AErUYCWjALx9RpJWYgfnEJdsQ1p7vhHi83oJLJE6HkugAjKD7YBAA)

Reply:
this wasn't working - still wasn't able to get TS to accept the `userAddress` const after the function call - so leaving as is for now